code/game/objects/items/weaponry.dm

```diff
@@ -55,6 +55,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "What are you standing around staring at this for? Get to killing!"
 	icon_state = "claymore"
 	inhand_icon_state = "claymore"
+	worn_icon = 'icons/mob/clothing/back.dmi'

```

Comment:
already handled by build_worn_icon. needs a new sprite instead, otherwise this is a bandaid fix.

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.